### PR TITLE
[ADVAPP-646]: Enhance the dropdown menu for profile information with an explicit Profile Settings link

### PR DESCRIPTION
**Pull request by payal-canyon**
### Ticket(s) or GitHub Issue

- https://canyongbs.atlassian.net/browse/ADVAPP-646

### Technical Description

> Enhance the dropdown menu for profile information to follow this order:
a. Icon Name
b. Mode Options
c. Profile Settings
d. Log Out

Please find the attachment below for reference:

image

### CODE REVIEW

**Review by danharrin — CHANGES_REQUESTED** [review]

File: app/Providers/Filament/AdminPanelProvider.php [hunk 1]
```diff
@@ -174,7 +174,9 @@ public function panel(Panel $panel): Panel
                 FilamentFullCalendarPlugin::make(),
             ])
             ->userMenuItems([
-                'profile' => MenuItem::make()
+                'profile' => MenuItem::make(),
+                'profile_settings' => MenuItem::make()
+                    ->label('Profile Settings')
```
Comment (mxm1070) [inline app/Providers/Filament/AdminPanelProvider.php:179]: Maybe add an icon like `heroicon-o-cog-6-tooth`.

**Review by mxm1070 — APPROVED** [review]

**Review by mxm1070 — CHANGES_REQUESTED** [review]
Maybe add an icon like `heroicon-o-cog-6-tooth`.

File: app/Providers/Filament/AdminPanelProvider.php [hunk 2]
```diff
@@ -174,7 +174,9 @@ public function panel(Panel $panel): Panel
                 FilamentFullCalendarPlugin::make(),
             ])
             ->userMenuItems([
-                'profile' => MenuItem::make()
+                'profile' => MenuItem::make(),
+                'profile_settings' => MenuItem::make()
```
[the remark this answers is not in the document]
Comment (payal-canyon) [reply app/Providers/Filament/AdminPanelProvider.php]: Done.

**Review by mxm1070 — APPROVED** [review]

File: app/Providers/Filament/AdminPanelProvider.php [hunk 3]
```diff
@@ -174,8 +174,10 @@ public function panel(Panel $panel): Panel
                 FilamentFullCalendarPlugin::make(),
             ])
             ->userMenuItems([
-                'profile' => MenuItem::make()
-                    ->url(fn () => EditProfile::getUrl()),
+                MenuItem::make()
+                    ->label('Profile Settings')
+                    ->url(fn () => EditProfile::getUrl())
+                    ->icon('heroicon-o-cog-6-tooth'),
```
Comment (danharrin) [inline app/Providers/Filament/AdminPanelProvider.php:180]: Based on the size of the icon I think it should be solid

```suggestion
                    ->icon('heroicon-s-cog-6-tooth'),
```

**Review by danharrin — CHANGES_REQUESTED** [review]

**Review by Orrison — APPROVED** [review]

**Comment by Orrison** [comment]
@danharrin and @mxm1070 please re-review.

**Review by mxm1070 — APPROVED** [review]

**Pull request merged**